packages/stream-metadata/.env.local.sample

```diff
@@ -2,8 +2,4 @@
 RIVER_ENV=local_multi
 RIVER_CHAIN_RPC_URL=http://localhost:8546
 # port that the server listens at. e.g. http://localhost:3002
-PORT=3002
-
-# IMPORTANT: disable tls cert validation in localhost env ONLY
-# in all other environments, comment out this line.
-NODE_TLS_REJECT_UNAUTHORIZED=0
```

Comment:
don't delete this for .env.local-sample. Otherwise, for local development, making a contract call to the local river chain will fail with 404 due to untrusted cert validation